entities/card_type.py

```diff
@@ -0,0 +1,6 @@
+from enum import Enum
+
+
+class CardType(Enum):
```

Comment:
change back to answer and question in the enum value

Reply:
What ev'